pythonFiles/tests/debug_adapter/test_ptvsd_folder_name.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+import sys
+
+from os import path
+
+try:
+    from unittest.mock import patch, mock_open
+
+    from packaging.tags import sys_tags
+    from ptvsd_folder_name import ptvsd_folder_name
+except:  # Python 2.7
+    print("Not importing anything for Python 2.7 since the test will be skipped.")
+
+
+ROOT = path.dirname(path.dirname(path.dirname(path.dirname(path.abspath(__file__)))))
+PYTHONFILES = path.join(ROOT, "pythonFiles", "lib", "python")
+
+
+def isPython37():
+    return sys.version_info.major == 3 and sys.version_info.minor == 7
```

Comment:
What happens when we go to 3.8, etc.?  I'd expect 3.7+ to be fine, in which case the comparison is simpler:

```suggestion
    return sys.version_info >= (3, 7)
```